doc/releases/migration-guide-3.7.rst

```diff
@@ -92,6 +92,10 @@ MbedTLS
   of other Kconfig options which are now named `CONFIG_MBEDTLS_SHA*`.
 * The `CONFIG_MBEDTLS_MAC_ALL_ENABLED` Kconfig option has been removed. Its equivalent is the
   combination of :kconfig:option:`CONFIG_MBEDTLS_HASH_ALL_ENABLED` and :kconfig:option:`CONFIG_MBEDTLS_CMAC`.
+* Symbols :kconfig:option:`CONFIG_MBEDTLS_MAC_MD4_ENABLED`,
+  :kconfig:option:`CONFIG_MBEDTLS_CIPHER_ARC4_ENABLED` and
+  :kconfig:option:`MBEDTLS_CIPHER_BLOWFISH_ENABLED` were removed because they
```

Comment:
```suggestion
  :kconfig:option:`CONFIG_MBEDTLS_CIPHER_BLOWFISH_ENABLED` were removed because they
```

btw since these Kconfig are gone anyways you may want to simply use double backticks instead of the `:kconfig:option:' role

Reply:
Thanks for the tip, I didn't thought about removing the `:kconfig:option:` part ;)